backend/src/SM.Service/Patterns/PatternsController.cs

```diff
@@ -11,6 +12,45 @@ namespace SM.Service.Patterns
     [Authorize, Route("api/patterns")]
     public class PatternsController : Controller
     {
+        [HttpGet, Route("getall")]
+        public async Task<IActionResult> GetAll()
+        {
+            var userId = User.GetUserId();
+            if (userId == null) return BadRequest();
+
+            var (patternsByOwnerProjection, _) = await Cluster.GetAsync(ActorKind.PatternsByOwnerProjection, ActorKind.PatternsByOwnerProjection);
+
+            var query = new GetPatternItems {RequestId = Guid.NewGuid().ToString(), OwnerId = userId, Skip = 0, Take = 100};
+
+            object response;
+
+            do
```

Comment:
This logic can be implemented cleaner. Maybe separate method will work best. Or even extension, something like `RequestRetry` with timeout, not sure. I would start with separate method with something like:

```csharp
while (true)
{
  if (some timeout condition) throw new TimeoutException();
  var response = await patternsByOwnerProjection.RequestAsync<object>(query, 10.Seconds());
  switch (response)
  {
    case PatternItems items:
      return items;
    case CatchingUp _:
      await Task.Delay(100);
      break;
    default:
      throw new Exception("Unknown response type."); // Or alike. Check what Proto.Actor throws in such case.
}
```